pkg/graph/resolvers/mto_category.go

```diff
@@ -138,3 +139,39 @@ func MTOSubcategoryGetByParentIDLoader(ctx context.Context, modelPlanID uuid.UUI
 	// return while adding an uncategorized record as well
 	return append(dbSubcategories, models.MTOUncategorizedSubcategoryFromArray(modelPlanID, &parentID, dbSubcategories)), nil
 }
+
+func MTOCategoryGetByID(ctx context.Context, id uuid.UUID) (*models.MTOCategory, error) {
+	return loaders.MTOCategory.ByID.Load(ctx, id)
+}
+
+// MTOCategoriesGetByID returns the subcategory and parent category information given an mto category id
+// It will return uncategorized for nil values
+// We are not doing a larger SQL call that would return both objects, as that
```

Comment:
Can you finish out this comment?

Reply:
Might also be worth adding the overall "flow" of this function as a comment, i.e.

> This function first fetches the category by the provided ID. Then, if that category is a parent category (and we have nothing else to fetch), returns that category. However, if the supplied ID points to a subcategory (i.e. it has a parent ID), then it also fetches the parent category information so both category AND subcategory are 
 returned as part of this resolver.

Reply:
Thanks for catching this Clay!